tests/Psr6StoreTest.php

```diff
@@ -225,7 +229,7 @@ public function testWriteStoresEntries()
 
         $entries = $cacheItem->get();
 
-        $this->assertInternalType('array', $entries, 'Entries are stored in cache.');
+        $this->assertTrue(is_array($entries), 'Entries are stored in cache.');
```

Comment:
This should be `\is_array()`. Normally, I have my CS fixer set up so that it automatically fixes those. Did I miss that here maybe or did you just not run the CS fixer? 😄

Reply:
No the ci didn't tell me todo this 😄 maybe you want to activate https://prettyci.com/ for this project.

Reply:
We may also just add the cs fixer to the travis job, but don't worry, I'll do that :)